wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/addtowallet/AddToWalletScreen.kt

```diff
@@ -218,4 +285,46 @@ private fun AddToWalletScreenWithIssuerDisplayDatas(
             }
         }
     }
+
+    // Scan Credential Offer
+    FilledTonalButton(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(8.dp),
+
+        colors = ButtonColors(
+            containerColor = MaterialTheme.colorScheme.primaryContainer,
+            contentColor = MaterialTheme.colorScheme.onPrimaryContainer,
+            disabledContainerColor = Color.Green,
+            disabledContentColor = Color.Green
+        ),
+
+        shape = RoundedCornerShape(8.dp),
+        onClick = {
+            onShowScanQrDialog.invoke()
```

Comment:
Why onShowScanQrDialog.invoke() instead of just onShowScanQrDialog() (or even onClick = onShowScanQrDialog)?

Reply:
`invoke` is used on lambdas so they're called like regular functions 
[invoke lambda doc](https://kotlinlang.org/docs/lambdas.html#invoking-a-function-type-instance)

I like your second approach a lot more thank you!! I've added the change!